src/components/FormInput/FormInput.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+export function FormInput({
+  id,
+  name,
+  placeholder,
+  type,
+  label,
+  defaultValue,
+  minLength,
+  maxLength,
+  pattern,
+  handleInput,
+}) {
+  return (
+    <label htmlFor={id} className="form__label">
+      {label}
+      <input
+        type={type}
+        className="field-input "
+        placeholder={placeholder}
+        id={id}
+        name={name}
+        defaultValue={defaultValue}
```

Comment:
`defaultValue` is used only with `useRef`. For controlled components you need to use only `value`